spec/requests/api/v1/users/fish/delete_fish_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "rails_helper"
+
+RSpec.describe "Fish", type: :request do
+  describe "DELETE /api/v1/users/:user_id/fish/:id" do
+    it "can delete a fish w/ valid params" do
+      user = create(:user)
+      fish = user.fish.create(species: "Walleye", weight: 1.1, length: 12.1)
+
+      headers = { "CONTENT_TYPE" => "application/json" }
+      delete "/api/v1/users/#{user.id}/fish/#{fish.id}", headers: headers
+
+      expect(response).to be_successful
+      expect(response.status).to eq(200)
```

Comment:
Should a successful delete return a 204 no content?

Reply:
I was thinking that, I noticed its a 200 in the lures delete as well, probably need to refactor both of these

Reply:
oh I see your below comment about the 204, got it